### PR TITLE
Cole Anthony

### PR DESCRIPTION
**Pull request by coleanthony1990**

### CODE REVIEW

File: section1/reflection.md [hunk 1]
```diff
@@ -2,18 +2,39 @@
...
 7. In your own words, explain what concatenation is.
 
+  Concatenation is when you combine variables to log them in the console is a specific order.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+  Rosetta Stone might use concatenation to give different examples of phrases in other languages.
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+  I'm having difficulty thinking of how variables are used. I know that they are, but it's hard for me to grasp the real life uses for them outside of these exercise. so I guess my question is how are variable actually used?
```
Comment (SageOfCode) [inline section1/reflection.md:40]: A common example could be getting a website visitor's name when they register on a website, and then displaying that name when they login.

File: section2/reflection.md [hunk 2]
```diff
@@ -2,14 +2,34 @@
...
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+logical operators such as && to determine if "condition" and "condition" are met and || to determine if "condition" or "condition" are met.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+I mostly get it but I'm not 100% on when to end a conditional with just "else."
```
Comment (SageOfCode) [inline section2/reflection.md:35]: For any specific situations that you want to cover, you will usually have an `if` or `else if`, `else` is usually used as to catch any other conditions that are not accounted for.

**Review by SageOfCode — COMMENTED** [review]
Great work here @coleanthony1990 !

- Your code looks good, and it looks like you were able to get a good feel for the topics discussed in mod 0.
- Your calendar looks great, it is good to see that you are blocking off times for things like lunch and walking your dog, that is super helpful for the full time program.
- Your letter to the instructors is well thought out.
- Good work on the mentor gist, that is an important relationship to build.
- Group work DTR looks good as well.

File: section2/reflection.md (hunk 2 above)
Comment (coleanthony1990) [reply section2/reflection.md:35]: Gotcha. It kind of seemed like this but I guess I wanted someone to exclusively say that. Thanks!

File: section1/reflection.md (hunk 1 above)
Comment (coleanthony1990) [reply section1/reflection.md:40]: Okay this makes sense. Thanks for example!

**Pull request closed without merging**